assets/app/view/game/corporation.rb

```diff
@@ -147,6 +149,59 @@ def render
         h('div.corp.card', { style: card_style, on: { click: select_corporation } }, children)
       end
 
+      def render_bidders
```

Comment:
Is there a way to combine this with render_bidders from the `Company` view?  I didn't see an appropriate file that both included, so I just copy/pasted the code with minor styling differences for padding and center text to match the Company look.